src/main/cli/settings.c

```diff
@@ -1473,7 +1473,7 @@ const clivalue_t valueTable[] = {
 #endif // end of #ifdef USE_OSD
 
 // PG_SYSTEM_CONFIG
-#if defined(STM32F4) || defined(STM32G4)
+#if defined(STM32F4) || defined(STM32G4) || defined(STM32H7)
```

Comment:
Can you please change this to USE_CONFIGURABLE_HSE or similar

Then add that to the relevant target.h

Reply:
Done